index.yaml

```diff
@@ -3436,26 +3339,7 @@ entries:
       catalog.cattle.io/requires-gvr: networking.istio.io.virtualservice/v1beta1
     apiVersion: v1
     appVersion: 0.8.0
-    created: "2021-06-16T21:05:17.464466-07:00"
-    description: The application deployment engine for Kubernetes
-    digest: 9170cd53f50cab95e89f18180bd2d50fd40bba30cae9af4468cacd33e43a03a0
-    home: https://rio.io
-    icon: https://charts.rancher.io/assets/logos/rio.svg
-    name: rio
-    urls:
-    - assets/rio/rio-100.0.0.tgz
-    version: 100.0.0
```

Comment:
100.0.0 is gone since its package dir is gone. 
The rest versions are from release-v2.6 branch.